static/sass/styles.scss

```diff
@@ -1814,3 +1814,10 @@ ol.p-stepped-list.no-full-stop
     scroll-margin-top: 100px;
   }
 }
+
+.channel-offer-table tbody tr td {
```

Comment:
Perhaps this could be extracted to a new file? There's `_pattern_distributor.scss`, not sure if it's the right place.

Reply:
Moved to `_pattern_distributor.scss`, thanks!